source/jormungandr/tests/kirin_realtime_tests.py

```diff
@@ -234,6 +255,17 @@ def _check_train_delay_disruption(dis):
         eq_(get_arrivals(new_base), ['20120614T080222', '20120614T080436'])
         eq_(get_used_vj(new_base), [['vjA'], []])
 
+        # we have one delayed departure
+        departures = self.query_region("stop_points/stop_point:stopB/departures?_current_datetime=20120614T0800", display=True)
+        assert len(departures['disruptions']) == 2
+        assert len(departures['departures']) == 2
+        assert departures['departures'][0]['stop_date_time']['departure_date_time'] == '20120614T080225'
+        # only one more disruption in base_schedule
```

Comment:
hum maybe it would be clearer to say that we have one more disruption not compared to the real_time, but compare to before the dirsruption no ?